vision/interface.py

```diff
@@ -1,7 +1,7 @@
 
 """The Environment class will manage the other objects in the environment"""
-
-
+import multiprocessing as mp 
+from functools import partial
 class Environment:
 
     #list of rectangles in environment
```

Comment:
Let's switch this comment to a docstring, I'm not sure if code editors will pick up on this.